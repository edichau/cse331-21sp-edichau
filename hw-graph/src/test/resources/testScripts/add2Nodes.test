# Tests that 2 nodes can be created

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2
