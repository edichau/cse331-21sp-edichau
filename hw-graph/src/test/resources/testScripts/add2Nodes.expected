# Tests that 2 nodes can be created

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1
